keyboards/planck/keymaps/rootiest/config.h

```diff
@@ -0,0 +1,134 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Sure, will do!

Reply:
Looks like it didn't get added/pushed

Reply:
Oh sorry, I did the keymap.c but not the config.h! Should be good now.